common-rendering/src/components/MetaData.tsx

```diff
@@ -0,0 +1,363 @@
+import { css } from "@emotion/react";
+import { headline, textSans } from "@guardian/src-foundations/typography";
+import { palette } from "@guardian/src-foundations";
+import { from, until } from "@guardian/src-foundations/mq";
+import { Pillar, Theme } from "@guardian/types";
+import {
+	culture,
+	lifestyle,
+	neutral,
+	news,
+	sport,
+	opinion,
+	text,
+} from "@guardian/src-foundations/palette";
+import { space } from "@guardian/src-foundations";
+
+export interface Authors {
+	id: string;
+	name: string;
+	contributor: boolean;
+}
+
+type BylineComponent = { token: string; tag: TagType } | string;
```

Comment:
Could we extract `{ token: string; tag: TagType }` to a new interface and name it in such a way as to make it slightly clearer what's happening here?

Reply:
This line is not being used at the moment, commented out.